reactioncommerce/reaction-admin
# feat: update to Meteor 1.10.2

## Changes
Update to Meteor 1.10.2 with Node 12.16.1 and version 3.5.7 of the `mongodb` library. This resolves potential vulnerabilities in NPM dependencies.

## Testing
1. Run `make dev-reaction-admin` to test the development build (happy path testing on admin UI).
2. Stop the admin service and delete the `docker-compose.override.yml` symlink.
3. Build production image with `docker build . -t reactioncommerce/admin:3.0.0-beta.local`
4. Temporarily update the image in `docker-compose.yml` to be `reactioncommerce/admin:3.0.0-beta.local`
5. Run `docker-compose up -d` and then verify that the admin UI works (happy path testing).
